example/src/screens/Examples/AwareScrollView/index.tsx

```diff
@@ -37,16 +39,52 @@ export default function AwareScrollView({ navigation }: Props) {
     });
   }, []);
 
+  const snapToOffsets = [125, 225, 325, 425];
+
   return (
     <>
       <KeyboardAwareScrollView
         testID="aware_scroll_view_container"
         bottomOffset={50}
         enabled={enabled}
+        snapToOffsets={snapToOffsetsEnabled ? snapToOffsets : undefined}
         disableScrollOnKeyboardHide={disableScrollOnKeyboardHide}
         style={styles.container}
         contentContainerStyle={styles.content}
       >
+        {snapToOffsetsEnabled && (
+          <>
+            {snapToOffsets.map((offset) => (
+              <View
+                key={offset}
+                style={{
+                  width: "100%",
+                  position: "absolute",
+                  top: offset,
+                }}
+              >
+                <View
+                  style={{
+                    flexDirection: "row",
+                    alignItems: "center",
+                    gap: 12,
+                  }}
+                >
+                  <Text>{offset}</Text>
+                  <View
+                    key={offset}
+                    style={{
+                      height: 2,
+                      flex: 1,
+                      backgroundColor: "black",
+                    }}
+                  ></View>
```

Comment:
I think linter/prettier will comply bout this, but do not worries, I'll approve this PR and it will run CI - we can see what failed and what not 👀

Reply:
Okay, it's not complaining, but can you use `<View />` syntax, please? 🙏